.gitpod.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gitpod/workspace-base
+FROM gitpod/workspace-node
 
 # Installing Node LTS
```

Comment:
Now we are working with `workspace-node`, you can remove `node` and `docker-compose` and `nvm` installion below